packages/api/src/beacon/routes/node.ts

```diff
@@ -19,19 +19,16 @@ import {WireFormat} from "../../utils/wireFormat.js";
 
 // See /packages/api/src/routes/index.ts for reasoning and instructions to add new routes
 
-export const byteListType = new ByteListType(100);
 export const NetworkIdentityType = new ContainerType(
   {
-    peerId: byteListType,
-    enr: byteListType,
-    p2pAddresses: ArrayOf(byteListType),
-    discoveryAddresses: ArrayOf(byteListType),
-    metadata: new ContainerType({
-      ...ssz.phase0.Metadata.fields,
-      // TODO: optional type does not really map to a field that can be `undefined` / missing
-      // the container would throw an error if JSON key is not defined, it rather maps to `type | null`
-      syncnets: new OptionalType(ssz.altair.Metadata.fields.syncnets),
```

Comment:
As noted in the comment, optional ssz type doesn't really map to a optional property but rather `type | null`, the container will throw an error if JSON key is not defined, we can't use it like that.

Noticed on unstable branch we also just use `ssz.altair.Metadata`, this means one of two things
- all clients return `syncnets` by now (I would assume this is the case)
- or nobody actually uses `@lodestar/api` to query this api (which might be the case as well)